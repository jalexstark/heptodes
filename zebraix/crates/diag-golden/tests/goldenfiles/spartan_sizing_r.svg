<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="350pt" viewBox="0 0 400 350">
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 40.910156 334.089844 L 40.910156 15.910156 L 359.089844 15.910156 L 359.089844 334.089844 Z M 40.910156 334.089844 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 40.910156 148.484375 L 359.089844 148.484375 M 40.910156 201.515625 L 359.089844 201.515625 M 40.910156 254.546875 L 359.089844 254.546875 M 40.910156 307.574219 L 359.089844 307.574219 M 40.910156 95.453125 L 359.089844 95.453125 M 40.910156 42.425781 L 359.089844 42.425781 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 279.546875 334.089844 L 279.546875 15.910156 M 226.515625 334.089844 L 226.515625 15.910156 M 173.484375 334.089844 L 173.484375 15.910156 M 120.453125 334.089844 L 120.453125 15.910156 M 67.425781 334.089844 L 67.425781 15.910156 M 332.574219 334.089844 L 332.574219 15.910156 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 335.375 148.484375 C 335.375 152.21875 329.777344 152.21875 329.777344 148.484375 C 329.777344 144.75 335.375 144.75 335.375 148.484375 Z M 229.316406 42.425781 C 229.316406 46.15625 223.714844 46.15625 223.714844 42.425781 C 223.714844 38.691406 229.316406 38.691406 229.316406 42.425781 Z M 229.316406 42.425781 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 329.644531 119.039062 L 335.507812 124.898438 M 335.507812 119.039062 L 329.644531 124.898438 Z M 250.101562 39.492188 L 255.960938 45.355469 M 255.960938 39.492188 L 250.101562 45.355469 Z M 255.960938 39.492188 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 332.574219 148.484375 L 332.574219 121.96875 M 226.515625 42.425781 L 253.03125 42.425781 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 282.34375 201.515625 C 282.34375 205.25 276.746094 205.25 276.746094 201.515625 C 276.746094 197.78125 282.34375 197.78125 282.34375 201.515625 Z M 176.285156 95.453125 C 176.285156 99.1875 170.683594 99.1875 170.683594 95.453125 C 170.683594 91.722656 176.285156 91.722656 176.285156 95.453125 Z M 176.285156 95.453125 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 276.613281 148.585938 L 282.476562 154.449219 M 282.476562 148.585938 L 276.613281 154.449219 Z M 220.550781 92.523438 L 226.414062 98.386719 M 226.414062 92.523438 L 220.550781 98.386719 Z M 226.414062 92.523438 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 279.546875 201.515625 L 279.546875 151.519531 M 173.484375 95.453125 L 223.480469 95.453125 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 229.316406 254.546875 C 229.316406 258.277344 223.714844 258.277344 223.714844 254.546875 C 223.714844 250.8125 229.316406 250.8125 229.316406 254.546875 Z M 123.253906 148.484375 C 123.253906 152.21875 117.65625 152.21875 117.65625 148.484375 C 117.65625 144.75 123.253906 144.75 123.253906 148.484375 Z M 123.253906 148.484375 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 223.585938 180.90625 L 229.445312 186.769531 M 229.445312 180.90625 L 223.585938 186.769531 Z M 188.230469 145.554688 L 194.09375 151.414062 M 194.09375 145.554688 L 188.230469 151.414062 Z M 194.09375 145.554688 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 226.515625 254.546875 L 226.515625 183.839844 M 120.453125 148.484375 L 191.160156 148.484375 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 176.285156 307.574219 C 176.285156 311.308594 170.683594 311.308594 170.683594 307.574219 C 170.683594 303.84375 176.285156 303.84375 176.285156 307.574219 Z M 70.222656 201.515625 C 70.222656 205.25 64.625 205.25 64.625 201.515625 C 64.625 197.78125 70.222656 197.78125 70.222656 201.515625 Z M 70.222656 201.515625 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 170.554688 145.554688 L 176.414062 151.414062 M 176.414062 145.554688 L 170.554688 151.414062 Z M 223.585938 198.585938 L 229.445312 204.445312 M 229.445312 198.585938 L 223.585938 204.445312 Z M 229.445312 198.585938 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(37%, 28%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 173.484375 307.574219 L 173.484375 148.484375 M 67.425781 201.515625 L 226.515625 201.515625 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 334.5625 140.953125 C 334.5625 144.6875 328.960938 144.6875 328.960938 140.953125 C 328.960938 137.21875 334.5625 137.21875 334.5625 140.953125 Z M 331.8125 131.550781 C 331.8125 135.285156 326.210938 135.285156 326.210938 131.550781 C 326.210938 127.816406 331.8125 127.816406 331.8125 131.550781 Z M 326.71875 120.347656 C 326.71875 124.078125 321.117188 124.078125 321.117188 120.347656 C 321.117188 116.613281 326.71875 116.613281 326.71875 120.347656 Z M 319.058594 107.691406 C 319.058594 111.425781 313.457031 111.425781 313.457031 107.691406 C 313.457031 103.957031 319.058594 103.957031 319.058594 107.691406 Z M 308.925781 94.265625 C 308.925781 97.996094 303.328125 97.996094 303.328125 94.265625 C 303.328125 90.53125 308.925781 90.53125 308.925781 94.265625 Z M 296.808594 80.992188 C 296.808594 84.726562 291.207031 84.726562 291.207031 80.992188 C 291.207031 77.257812 296.808594 77.257812 296.808594 80.992188 Z M 283.535156 68.875 C 283.535156 72.605469 277.9375 72.605469 277.9375 68.875 C 277.9375 65.140625 283.535156 65.140625 283.535156 68.875 Z M 270.109375 58.742188 C 270.109375 62.472656 264.507812 62.472656 264.507812 58.742188 C 264.507812 55.007812 270.109375 55.007812 270.109375 58.742188 Z M 257.453125 51.082031 C 257.453125 54.816406 251.855469 54.816406 251.855469 51.082031 C 251.855469 47.347656 257.453125 47.347656 257.453125 51.082031 Z M 246.25 45.988281 C 246.25 49.722656 240.648438 49.722656 240.648438 45.988281 C 240.648438 42.257812 246.25 42.257812 246.25 45.988281 Z M 236.847656 43.238281 C 236.847656 46.972656 231.246094 46.972656 231.246094 43.238281 C 231.246094 39.503906 236.847656 39.503906 236.847656 43.238281 Z M 236.847656 43.238281 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 281.574219 188.75 C 281.574219 192.484375 275.972656 192.484375 275.972656 188.75 C 275.972656 185.015625 281.574219 185.015625 281.574219 188.75 Z M 279.136719 175.628906 C 279.136719 179.363281 273.539062 179.363281 273.539062 175.628906 C 273.539062 171.898438 279.136719 171.898438 279.136719 175.628906 Z M 274.898438 162.476562 C 274.898438 166.207031 269.296875 166.207031 269.296875 162.476562 C 269.296875 158.742188 274.898438 158.742188 274.898438 162.476562 Z M 268.796875 149.648438 C 268.796875 153.378906 263.199219 153.378906 263.199219 149.648438 C 263.199219 145.914062 268.796875 145.914062 268.796875 149.648438 Z M 260.875 137.535156 C 260.875 141.269531 255.273438 141.269531 255.273438 137.535156 C 255.273438 133.804688 260.875 133.804688 260.875 137.535156 Z M 251.28125 126.519531 C 251.28125 130.253906 245.679688 130.253906 245.679688 126.519531 C 245.679688 122.785156 251.28125 122.785156 251.28125 126.519531 Z M 240.265625 116.925781 C 240.265625 120.65625 234.664062 120.65625 234.664062 116.925781 C 234.664062 113.191406 240.265625 113.191406 240.265625 116.925781 Z M 228.152344 109.003906 C 228.152344 112.734375 222.554688 112.734375 222.554688 109.003906 C 222.554688 105.269531 228.152344 105.269531 228.152344 109.003906 Z M 215.324219 102.902344 C 215.324219 106.632812 209.726562 106.632812 209.726562 102.902344 C 209.726562 99.167969 215.324219 99.167969 215.324219 102.902344 Z M 202.167969 98.660156 C 202.167969 102.394531 196.570312 102.394531 196.570312 98.660156 C 196.570312 94.929688 202.167969 94.929688 202.167969 98.660156 Z M 189.050781 96.226562 C 189.050781 99.957031 183.449219 99.957031 183.449219 96.226562 C 183.449219 92.492188 189.050781 92.492188 189.050781 96.226562 Z M 189.050781 96.226562 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 228.578125 237.605469 C 228.578125 241.339844 222.980469 241.339844 222.980469 237.605469 C 222.980469 233.871094 228.578125 233.871094 228.578125 237.605469 Z M 226.367188 222.136719 C 226.367188 225.871094 220.769531 225.871094 220.769531 222.136719 C 220.769531 218.40625 226.367188 218.40625 226.367188 222.136719 Z M 222.6875 208.144531 C 222.6875 211.878906 217.085938 211.878906 217.085938 208.144531 C 217.085938 204.410156 222.6875 204.410156 222.6875 208.144531 Z M 217.53125 195.621094 C 217.53125 199.355469 211.929688 199.355469 211.929688 195.621094 C 211.929688 191.890625 217.53125 191.890625 217.53125 195.621094 Z M 210.902344 184.574219 C 210.902344 188.308594 205.300781 188.308594 205.300781 184.574219 C 205.300781 180.839844 210.902344 180.839844 210.902344 184.574219 Z M 202.800781 175 C 202.800781 178.734375 197.199219 178.734375 197.199219 175 C 197.199219 171.265625 202.800781 171.265625 202.800781 175 Z M 193.226562 166.898438 C 193.226562 170.632812 187.625 170.632812 187.625 166.898438 C 187.625 163.164062 193.226562 163.164062 193.226562 166.898438 Z M 182.175781 160.269531 C 182.175781 164.003906 176.578125 164.003906 176.578125 160.269531 C 176.578125 156.535156 182.175781 156.535156 182.175781 160.269531 Z M 169.65625 155.113281 C 169.65625 158.847656 164.054688 158.847656 164.054688 155.113281 C 164.054688 151.378906 169.65625 151.378906 169.65625 155.113281 Z M 155.660156 151.429688 C 155.660156 155.164062 150.0625 155.164062 150.0625 151.429688 C 150.0625 147.699219 155.660156 147.699219 155.660156 151.429688 Z M 140.195312 149.222656 C 140.195312 152.953125 134.59375 152.953125 134.59375 149.222656 C 134.59375 145.488281 140.195312 145.488281 140.195312 149.222656 Z M 140.195312 149.222656 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 65%)" stroke-opacity="1" stroke-miterlimit="10" d="M 175.667969 276.34375 C 175.667969 280.078125 170.066406 280.078125 170.066406 276.34375 C 170.066406 272.613281 175.667969 272.613281 175.667969 276.34375 Z M 174.097656 256.1875 C 174.097656 259.917969 168.496094 259.917969 168.496094 256.1875 C 168.496094 252.453125 174.097656 252.453125 174.097656 256.1875 Z M 171.773438 242.132812 C 171.773438 245.867188 166.171875 245.867188 166.171875 242.132812 C 166.171875 238.402344 171.773438 238.402344 171.773438 242.132812 Z M 168.710938 231.816406 C 168.710938 235.550781 163.109375 235.550781 163.109375 231.816406 C 163.109375 228.085938 168.710938 228.085938 168.710938 231.816406 Z M 164.832031 223.964844 C 164.832031 227.699219 159.230469 227.699219 159.230469 223.964844 C 159.230469 220.230469 164.832031 220.230469 164.832031 223.964844 Z M 159.96875 217.832031 C 159.96875 221.566406 154.367188 221.566406 154.367188 217.832031 C 154.367188 214.097656 159.96875 214.097656 159.96875 217.832031 Z M 153.835938 212.96875 C 153.835938 216.703125 148.234375 216.703125 148.234375 212.96875 C 148.234375 209.234375 153.835938 209.234375 153.835938 212.96875 Z M 145.980469 209.089844 C 145.980469 212.824219 140.382812 212.824219 140.382812 209.089844 C 140.382812 205.359375 145.980469 205.359375 145.980469 209.089844 Z M 135.664062 206.027344 C 135.664062 209.761719 130.066406 209.761719 130.066406 206.027344 C 130.066406 202.296875 135.664062 202.296875 135.664062 206.027344 Z M 121.613281 203.703125 C 121.613281 207.433594 116.015625 207.433594 116.015625 203.703125 C 116.015625 199.96875 121.613281 199.96875 121.613281 203.703125 Z M 101.453125 202.132812 C 101.453125 205.867188 95.855469 205.867188 95.855469 202.132812 C 95.855469 198.398438 101.453125 198.398438 101.453125 202.132812 Z M 101.453125 202.132812 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 332.574219 148.484375 C 332.574219 131.003906 317.320312 104.300781 294.007812 80.992188 C 270.699219 57.679688 243.996094 42.425781 226.515625 42.425781 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 279.546875 201.515625 C 279.546875 142.941406 232.058594 95.453125 173.484375 95.453125 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 226.515625 254.546875 C 226.515625 183.839844 191.160156 148.484375 120.453125 148.484375 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 173.484375 307.574219 L 173.445312 298.617188 L 173.328125 290.699219 L 173.148438 283.648438 L 172.910156 277.335938 L 172.617188 271.644531 L 172.277344 266.492188 L 171.886719 261.808594 L 171.453125 257.53125 L 170.976562 253.609375 L 170.453125 250 L 169.894531 246.671875 L 169.289062 243.589844 L 168.644531 240.730469 L 167.957031 238.070312 L 167.226562 235.59375 L 166.449219 233.277344 L 165.628906 231.113281 L 164.761719 229.082031 L 163.847656 227.171875 L 162.878906 225.378906 L 161.855469 223.691406 L 160.777344 222.097656 L 159.640625 220.59375 L 158.4375 219.175781 L 157.167969 217.832031 L 155.824219 216.5625 L 154.40625 215.359375 L 152.902344 214.222656 L 151.308594 213.144531 L 149.621094 212.121094 L 147.828125 211.152344 L 145.917969 210.238281 L 143.886719 209.371094 L 141.722656 208.550781 L 139.40625 207.773438 L 136.929688 207.042969 L 134.269531 206.355469 L 131.410156 205.710938 L 128.328125 205.105469 L 125 204.546875 L 121.390625 204.023438 L 117.46875 203.546875 L 113.191406 203.113281 L 108.507812 202.722656 L 103.355469 202.382812 L 97.664062 202.089844 L 91.351562 201.851562 L 84.300781 201.671875 L 76.382812 201.554688 "/>
</svg>
