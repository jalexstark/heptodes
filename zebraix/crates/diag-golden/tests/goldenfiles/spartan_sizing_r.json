{
  "sizing_scheme": "SquareCenter",
  "base_height": 350.0,
  "axes_range": [
    -4.5,
    -3.5,
    1.5,
    2.5
  ],
  "padding": [
    0.05
  ],
  "drawables": [
    {
      "drawable": {
        "Polyline": {
          "line_closure_choice": "Closed",
          "locations": [
            [
              -4.5,
              -3.5
            ],
            [
              -4.5,
              2.5
            ],
            [
              1.5,
              2.5
            ],
            [
              1.5,
              -3.5
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              -4.5,
              0.0
            ]
          ],
          "end": [
            [
              1.5,
              0.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -0.0,
              -1.0
            ],
            [
              -0.0,
              -2.0
            ],
            [
              -0.0,
              -3.0
            ],
            [
              0.0,
              1.0
            ],
            [
              0.0,
              2.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              0.0,
              -3.5
            ]
          ],
          "end": [
            [
              0.0,
              2.5
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -1.0,
              -0.0
            ],
            [
              -2.0,
              -0.0
            ],
            [
              -3.0,
              -0.0
            ],
            [
              -4.0,
              -0.0
            ],
            [
              1.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "YellowBrown",
          "centers": [
            [
              1.0,
              0.0
            ],
            [
              -1.0,
              2.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "color_choice": "YellowBrown",
          "centers": [
            [
              1.0,
              0.5
            ],
            [
              -0.5,
              2.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "color_choice": "YellowBrown",
          "start": [
            [
              1.0,
              0.0
            ],
            [
              -1.0,
              2.0
            ]
          ],
          "end": [
            [
              1.0,
              0.5
            ],
            [
              -0.5,
              2.0
            ]
          ]
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              0.9846449136276391,
              0.14203454894433773
            ],
            [
              0.9327731092436973,
              0.31932773109243695
            ],
            [
              0.8367346938775508,
              0.5306122448979591
            ],
            [
              0.6923076923076922,
              0.769230769230769
            ],
            [
              0.5012468827930173,
              1.022443890274314
            ],
            [
              0.27272727272727254,
              1.2727272727272725
            ],
            [
              0.022443890274314003,
              1.5012468827930172
            ],
            [
              -0.23076923076923098,
              1.692307692307692
            ],
            [
              -0.469387755102041,
              1.8367346938775506
            ],
            [
              -0.6806722689075633,
              1.9327731092436973
            ],
            [
              -0.8579654510556624,
              1.984644913627639
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Arc": {
          "color_choice": "Green",
          "angle_range": [
            -1.1863995522992574,
            1.1863995522992574
          ],
          "center": [
            -0.16363636363636377,
            0.8363636363636362
          ],
          "transform": [
            0.4363636363636363,
            0.4363636363636363,
            -1.0787197799411874,
            1.0787197799411874
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "YellowBrown",
          "centers": [
            [
              0.0,
              -1.0
            ],
            [
              -2.0,
              1.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "color_choice": "YellowBrown",
          "centers": [
            [
              0.0,
              -0.05719095841793653
            ],
            [
              -1.0571909584179364,
              1.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "color_choice": "YellowBrown",
          "start": [
            [
              0.0,
              -1.0
            ],
            [
              -2.0,
              1.0
            ]
          ],
          "end": [
            [
              0.0,
              -0.05719095841793653
            ],
            [
              -1.0571909584179364,
              1.0
            ]
          ]
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              -0.014539496081668655,
              -0.7592790258819029
            ],
            [
              -0.06047582168733081,
              -0.5118955421832686
            ],
            [
              -0.14042339787513938,
              -0.26381058087625436
            ],
            [
              -0.2554791617945659,
              -0.021916647178263315
            ],
            [
              -0.40486657827068195,
              0.20646150658946344
            ],
            [
              -0.5857864376269049,
              0.4142135623730952
            ],
            [
              -0.7935384934105366,
              0.5951334217293182
            ],
            [
              -1.0219166471782632,
              0.7445208382054342
            ],
            [
              -1.2638105808762543,
              0.8595766021248609
            ],
            [
              -1.5118955421832687,
              0.9395241783126693
            ],
            [
              -1.7592790258819024,
              0.9854605039183315
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Arc": {
          "color_choice": "Green",
          "angle_range": [
            -0.7853981633974481,
            0.7853981633974481
          ],
          "center": [
            -2.0000000000000013,
            -1.0000000000000002
          ],
          "transform": [
            1.4142135623730963,
            1.4142135623730956,
            -1.4142135623730954,
            1.4142135623730954
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "YellowBrown",
          "centers": [
            [
              -1.0,
              -2.0
            ],
            [
              -3.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "color_choice": "YellowBrown",
          "centers": [
            [
              -1.0,
              -0.6666666666666667
            ],
            [
              -1.6666666666666667,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "color_choice": "YellowBrown",
          "start": [
            [
              -1.0,
              -2.0
            ],
            [
              -3.0,
              0.0
            ]
          ],
          "end": [
            [
              -1.0,
              -0.6666666666666667
            ],
            [
              -1.6666666666666667,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              -1.0138888888888888,
              -1.6805555555555554
            ],
            [
              -1.0555555555555556,
              -1.3888888888888888
            ],
            [
              -1.125,
              -1.1249999999999998
            ],
            [
              -1.2222222222222223,
              -0.8888888888888888
            ],
            [
              -1.3472222222222223,
              -0.6805555555555556
            ],
            [
              -1.5,
              -0.5
            ],
            [
              -1.6805555555555556,
              -0.34722222222222227
            ],
            [
              -1.8888888888888888,
              -0.22222222222222232
            ],
            [
              -2.125,
              -0.1250000000000001
            ],
            [
              -2.388888888888889,
              -0.05555555555555569
            ],
            [
              -2.6805555555555554,
              -0.01388888888888909
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Cubic": {
          "color_choice": "Green",
          "x": [
            -1.0,
            -1.0,
            -1.6666666666666667,
            -3.0
          ],
          "y": [
            -1.9999999999999998,
            -0.6666666666666665,
            -8.326672684688674e-17,
            -2.7755575615628914e-16
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "YellowBrown",
          "centers": [
            [
              -2.0,
              -3.0
            ],
            [
              -4.0,
              -1.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "color_choice": "YellowBrown",
          "centers": [
            [
              -2.0,
              0.0
            ],
            [
              -1.0,
              -1.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "color_choice": "YellowBrown",
          "start": [
            [
              -2.0,
              -3.0
            ],
            [
              -4.0,
              -1.0
            ]
          ],
          "end": [
            [
              -2.0,
              0.0
            ],
            [
              -1.0,
              -1.0
            ]
          ]
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "color_choice": "Blue",
          "centers": [
            [
              -2.011661807580175,
              -2.4110787172011663
            ],
            [
              -2.0412371134020617,
              -2.0309278350515463
            ],
            [
              -2.0851063829787235,
              -1.7659574468085106
            ],
            [
              -2.142857142857143,
              -1.5714285714285714
            ],
            [
              -2.2159827213822894,
              -1.4233261339092873
            ],
            [
              -2.307692307692308,
              -1.307692307692308
            ],
            [
              -2.4233261339092875,
              -1.2159827213822896
            ],
            [
              -2.571428571428571,
              -1.1428571428571428
            ],
            [
              -2.7659574468085104,
              -1.0851063829787233
            ],
            [
              -3.0309278350515463,
              -1.041237113402062
            ],
            [
              -3.4110787172011663,
              -1.0116618075801749
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Polyline": {
          "color_choice": "Green",
          "locations": [
            [
              -2.0,
              -3.0
            ],
            [
              -2.000762631077216,
              -2.8310772163965683
            ],
            [
              -2.002919708029197,
              -2.681751824817518
            ],
            [
              -2.0063102541630147,
              -2.5488168273444347
            ],
            [
              -2.0108108108108107,
              -2.4297297297297296
            ],
            [
              -2.016326530612245,
              -2.3224489795918366
            ],
            [
              -2.022784810126582,
              -2.2253164556962024
            ],
            [
              -2.030130668716372,
              -2.1369715603382016
            ],
            [
              -2.038323353293413,
              -2.0562874251497005
            ],
            [
              -2.0473338203067932,
              -1.9823228634039443
            ],
            [
              -2.0571428571428574,
              -1.9142857142857144
            ],
            [
              -2.0677396780965713,
              -1.8515045486354096
            ],
            [
              -2.0791208791208793,
              -1.7934065934065937
            ],
            [
              -2.0912896691424714,
              -1.7395003376097231
            ],
            [
              -2.104255319148936,
              -1.6893617021276595
            ],
            [
              -2.118032786885246,
              -1.6426229508196724
            ],
            [
              -2.1326424870466325,
              -1.5989637305699484
            ],
            [
              -2.148110185778347,
              -1.5581037796284434
            ],
            [
              -2.1644670050761423,
              -1.5197969543147207
            ],
            [
              -2.1817495280050347,
              -1.483826305852738
            ],
            [
              -2.2,
              -1.45
            ],
            [
              -2.219266625233064,
              -1.4181479179614669
            ],
            [
              -2.23960396039604,
              -1.3881188118811882
            ],
            [
              -2.2610734114743987,
              -1.3597779148673659
            ],
            [
              -2.2837438423645318,
              -1.3330049261083745
            ],
            [
              -2.307692307692308,
              -1.307692307692308
            ],
            [
              -2.3330049261083743,
              -1.2837438423645322
            ],
            [
              -2.3597779148673657,
              -1.2610734114743984
            ],
            [
              -2.388118811881188,
              -1.2396039603960396
            ],
            [
              -2.418147917961467,
              -1.2192666252330642
            ],
            [
              -2.45,
              -1.2000000000000002
            ],
            [
              -2.483826305852738,
              -1.1817495280050347
            ],
            [
              -2.5197969543147205,
              -1.164467005076142
            ],
            [
              -2.558103779628443,
              -1.1481101857783471
            ],
            [
              -2.598963730569948,
              -1.132642487046632
            ],
            [
              -2.642622950819672,
              -1.118032786885246
            ],
            [
              -2.6893617021276595,
              -1.1042553191489362
            ],
            [
              -2.7395003376097233,
              -1.0912896691424714
            ],
            [
              -2.793406593406593,
              -1.079120879120879
            ],
            [
              -2.8515045486354094,
              -1.0677396780965709
            ],
            [
              -2.914285714285714,
              -1.0571428571428572
            ],
            [
              -2.9823228634039443,
              -1.0473338203067932
            ],
            [
              -3.0562874251497005,
              -1.038323353293413
            ],
            [
              -3.136971560338201,
              -1.030130668716372
            ],
            [
              -3.2253164556962024,
              -1.0227848101265822
            ],
            [
              -3.322448979591837,
              -1.0163265306122449
            ],
            [
              -3.42972972972973,
              -1.010810810810811
            ],
            [
              -3.5488168273444343,
              -1.006310254163015
            ],
            [
              -3.681751824817518,
              -1.0029197080291972
            ],
            [
              -3.8310772163965683,
              -1.0007626310772164
            ]
          ]
        }
      }
    }
  ]
}
